WD: MongoDB project (#2359)
writethedocs/www: docs/conf/portland/2025/writing-day.md

The [Write the Docs Salary Survey](https://writethedocs.org/surveys/) has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations.

Our goal for Writing Day is to improve the clarity, inclusivity, neutrality and relevance of the [questions, instructions, and general flow of the survey](https://github.com/writethedocs/salary-survey?tab=readme-ov-file#write-the-docs-annual-documentation-salary-survey).

### Research the Docs!

Project organizer: Kyle Rollins, he/him. Project session: All day.

Join the MongoDB docs and design teams for a dual-purpose documentation research project. 

*What is this?* 

* An opportunity to work with a product designer on a research activity.
* Learn how to give qualitative feedback using MongoDB documentation.

*Why participate?*

* Join us to experience professional product research in action.

> * Leave with the knowledge and materials to facilitate youur own documentation research project.

Reviewer: ```suggestion
* Leave with the knowledge and materials to facilitate your own documentation research project.
```

> Your participation helps MongoDB better understand our own docs’ strengths and weaknesses. It also gives us the opportunity to share how you can facilitate your own documentation research project. We appreciate your help in making our documentation the best that it can be.

Reviewer: This is on WTD page, so the 'our' probably needs rewriting?

Reviewer 2: Ah, good point! In the context of the WTD Portland page, this could be a bit confusing. I left an alternative in a suggestion below.

Author: Perfect, I'll get those in and then we'll work on getting it live. Thanks!

> Your participation helps MongoDB better understand our own docs’ strengths and weaknesses. It also gives us the opportunity to share how you can facilitate your own documentation research project. We appreciate your help in making our documentation the best that it can be.

Reviewer 2: ```suggestion
Your participation helps the MongoDB team better understand the MongoDB docs’ strengths and weaknesses. It also gives MongoDB team the opportunity to share how you can facilitate your own documentation research project. We appreciate your help in making the MongoDB documentation the best that it can be.
```
